.golangci.yaml

```diff
@@ -0,0 +1,97 @@
+run:
+  timeout: 5m
+linters:
+  enable:
+    # region General
+
+    # Add depguard to prevent adding additional dependencies. This is a client library, we really don't want
+    # additional dependencies.
+    - depguard
+    # Prevent improper directives in go.mod.
+    - gomoddirectives
+    # Prevent improper nolint directives.
+    - nolintlint
+
+    # endregion
+
+
+    # region Code Quality and Comments
+
+    # Inspect source code for potential security problems. This check has a fairly high false positive rate,
+    # comment with // nolint:gosec where not relevant.
+    - gosec
+    # Complain about deeply nested if cases.
+    - nestif
+    # Prevent naked returns in long functions.
+    - nakedret
+    # Make Go code more readable.
+    - gocritic
+    # We don't want hidden global scope, so disallow global variables. You can disable this with
+    # Check if comments end in a period. This helps prevent incomplete comment lines, such as half-written sentences.
+    - godot
```

Comment:
Line 29 looks like an orphan.  Should it be removed, or should it be completed with a linter entry added after it?